kentik_api_library/kentik_api/requests_payload/conversions.py

```diff
@@ -0,0 +1,99 @@
+import json
```

Comment:
Nit: imports groups

Reply:
?

Reply:
In other files we group imports into 3 groups: stdlib, external, internal imports. I can see one group here

Reply:
Yeah, ulitimately we need tools to check and format those imports. It is a waste of time to do it by hand.